tests/test_update_metrics.py

```diff
@@ -0,0 +1,53 @@
+import csv
+import os
+import shutil
+import tempfile
+import unittest
+import src.tasks.tasks as tasks
+import torch
+
+
+class TestUpdateMetricsAccuracy(unittest.TestCase):
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+        self.path = os.path.join(self.temp_dir, "temp_dataset.tsv")
+        self.task = tasks.WiCTask("", 100, "winograd", tokenizer_name="MosesTokenizer")
+        predictions = torch.Tensor(
```

Comment:
I'll decrease it to 2 examples per batch - I just wanted to make sure there was more than 1 per batch since there can be edge cases that can break dimension-wise with only 1 per batch. 
And the predictions are 4d vectors rather than 2d because the predictions are given in 1-hot vector form for each batch.

Reply:
Add a comment explaining what the three axes are?

Reply:
In response to your point about batch size; also run a test to make sure we're catching batch size = 1 bugs (either here or in a separate test) ?